runtime/syntax/vim.vim

```diff
@@ -19,14 +19,14 @@ syn cluster vimCommentGroup	contains=vimTodo,@Spell
 
 " Special and plugin vim commands {{{2
 syn match   vimCommand contained	"\<z[-+^.=]\="
-syn keyword vimOnlyCommand contained	fix[del] sh[ell] P[rint]
+syn keyword vimOnlyCommand contained fix[del] sh[ell] te[aroff] P[rint]
```

Comment:
It's probably best to keep the Tab before `fix`. I guess you've been exposed way too much to my stuff, @Pyrohh ;)